.circleci/README.md

```diff
@@ -0,0 +1,14 @@
+CI Notes
+----------------
+
+These are the scripts that Circle CI uses to run the tests during a PR.
+
+Note: This uses the Chipyard `ucb-bar/chipyard:dev` branch to run CI (or other similar commits).
+
+Note: CircleCI env. var. refers to adding a environment variable by navigating to the "Project Settings" in the web UI and adding
+an environment variable in the "Environment Variables" tab.
+
+Note: This uses `$SERVER` and `$CI_DIR` which is given in the CircleCI env. var. setup to specify a server to build on.
+To change these variables you must change the project settings of CircleCI.
```

Comment:
Since these are just unit tests, is it necessary to build on a remote server? 
I think minimizing the amount of stuff which depends on that remote would reduce the pain if that server ever needs to be updated.

Reply:
For now, I think that is fine, I was just reusing the infrastructure so that people could extend this in the future.